src/ApiHandler.ts

```diff
@@ -232,6 +232,9 @@ export default class ApiHandler {
 			}
 		}
 
+		const header = await api.derive.chain.getHeader(hash);
```

Comment:
Can you not pull the author out of the header [here](https://github.com/paritytech/substrate-api-sidecar/blob/60face540ed5e85254000e78a2557ab516a70dd3/src/ApiHandler.ts#L53)?

Even if not, I think it'd make more sense to put this call [here](https://github.com/paritytech/substrate-api-sidecar/blob/60face540ed5e85254000e78a2557ab516a70dd3/src/ApiHandler.ts#L48)

Reply:
Good looking out here (I totally double-checked), but no I can't get the author ID from _that_ header. Here are the attributes of that header:
```
/** @name Header */
export interface Header extends Struct {
    readonly parentHash: Hash;
    readonly number: Compact<BlockNumber>;
    readonly stateRoot: Hash;
    readonly extrinsicsRoot: Hash;
    readonly digest: Digest;
}
```

I followed [the documented steps from the official Polkadot JS  website](https://polkadot.js.org/api/cookbook/blocks.html#how-do-i-extract-the-block-author). Agree that the placement you suggest is better; all things being equal I usually just add new stuff "to the end" but in this context you're definitely right :+1:

Reply:
Interesting, I do wonder if it's possible to get author from `digest`

Reply:
It looks like `api.derive.chain.getHeader` allows you to extract the author from `digest.logs` with some logic that is dependent on the runtime engine. I think we could probably replicate that logic here if we really wanted to avoid the extra RPC call. I believe we could stick [this exact logic in](https://github.com/polkadot-js/api/blob/master/packages/api-derive/src/type/HeaderExtended.ts#L28). Not sure how great of an idea code duplication is though since if that ever gets changed we would have to change it here, but I think its worth considering for performance

Reply:
This sounds like premature optimization to me. I believe that this naive solution will work in the short-term. Once we get some benchmarking in place, we can evaluate possible bottlenecks and optimize where needed. My $0.02.